src/newrelic/integration/test.go

```diff
@@ -513,26 +513,32 @@ func (t *Test) comparePhpPackages(harvest *newrelic.Harvest) {
 	var test_package_name_only bool = false
 
 	if nil != t.phpPackagesConfig {
-		expectedPkgsCollection, err := NewPhpPackagesCollection(t.Path, t.phpPackagesConfig)
-		if nil != err {
-			t.Fatal(err)
-			return
-		}
-		test_package_name_only = expectedPkgsCollection.config.package_name_only
-		if test_package_name_only {
-			t.AddNote("Tested package name only")
-		}
-		expectedPackages, err = expectedPkgsCollection.GatherInstalledPackages()
-		if nil != err {
-			t.Fatal(err)
-			return
+
+		expect_null_pkgs := "null" == string(t.phpPackagesConfig)
```

Comment:
I would suggest using camelCase for this variable name rather than underscores to maintain Go [best practices](https://go.dev/doc/effective_go#mixed-caps) (although I recognize this has been historically violated in the daemon code)

Reply:
I made a [commit](https://github.com/newrelic/newrelic-php-agent/pull/827/commits/1ac0d2da1148414f8460ec067be15c5739a9edad) to PR #827 which refactors the introduced variables by all of the code for `EXPECT_PHP_PACKAGES` to follow Go coding style.

Reply:
This [commit](https://github.com/newrelic/newrelic-php-agent/pull/826/commits/0c2ef65713a469769d2fe0b4871e9899b799cc74) fixes the style issue in this PR.